backend/svirt.pm

```diff
@@ -309,4 +309,8 @@ sub stop_serial_grab ($self, @) {
     return;
 }
 
+sub get_sleep_on_here_script_input ($self, $args) {
+    ($bmwqemu::vars{VIRSH_VMM_FAMILY} // '') =~ qr/^hyperv|vmware$/ ? 0.8 : 0;
```

Comment:
I *still* consider this a bad idea. My statement about "sleep" statement is not handled in a better way through this approach. If you *still* want to go ahead please add a comment here explaining why this is necessary

Reply:
ok, I mean we could also use this `wait_still_screen` thing. Which is in the end a hidden sleep for 1s. I did a test last night containing the following path. And I didn't notice a error on 2800 iterations.

diff --git a/distribution.pm b/distribution.pm
index a15132a0..adbe4538 100644
--- a/distribution.pm
+++ b/distribution.pm
@@ -279,9 +279,11 @@ sub script_output {
         testapi::wait_serial("$marker-0-", quiet => $args{quiet});
     }
     elsif ($args{type_command}) {
-        my $cat = "cat - > $script_path;\n";
+        my $cat = "cat - > $script_path;";
         testapi::type_string($cat);
-        sleep testapi::backend_get_sleep_on_here_script_input();
+        #        sleep testapi::backend_get_sleep_on_here_script_input();
+        my %args = testapi::backend_get_sleep_on_here_script_input() ? (similarity_level => 30, wait_still_screen => 1) : ();
+        testapi::type_string("\n", %args);
         testapi::type_string($script . "\n", timeout => $args{timeout});
         testapi::send_key('ctrl-d');
     }
```

The stats of one `script_output('w')` call of `wait_still_screen=>1` code (http://kepler.suse.cz/t8981)
```
Loop: 100
Allover: 906.873669862747
Avg: 9.06101491212844
Current:9.09190988540649
```

Compared to the sleep variation of the PR: (http://kepler.suse.cz/tests/8963#step/clemix/302)
```
Loop: 100
Allover: 838.612976074219
Avg: 8.37837042331696
Current:8.38157916069031
```

Compared to the current master (http://kepler.suse.cz/tests/8932#step/clemix/302)
```
Loop: 100
Allover: 757.928626060486
Avg: 7.57161826133728
Current:7.56396198272705
```

As you can see, the sleep is actually the better performing, but for me it is more important to get this fix for hyperv and vmware then doing one or the other way of slow down the hit level in that area.

Reply:
Yes, `wait_still_screen` is a hidden sleep. As I stated, if you *still* want to go ahead please add a comment here explaining why this is necessary if https://github.com/os-autoinst/os-autoinst/pull/1809#discussion_r754640715 does not help and you can't find a better solution that is not a sleep.